### ISSUE TITLE
java.lang.ClassCastException: android.view.ContextThemeWrapper cannot be cast to android.app.Activity

### ISSUE THREAD

**Issue by nomanbiniqbal** [oliveiradev/image-zoom#3]
> java.lang.ClassCastException: android.view.ContextThemeWrapper cannot be cast to android.app.Activity

**Comment by oliveiradev** [comment]
You can describe this problem to me @nomanbiniqbal ?

**Comment by nomanbiniqbal** [comment]
Hi 
Thanks for your response .
My Base is AppCompatActivity 
and i m using this inside following code just replace image view with your email and you will be able to reproduced this issue. 
following code is from example "collepsable toolbar with image"

`
        <android.support.design.widget.AppBarLayout
            android:id="@+id/app_bar"
            android:layout_width="match_parent"
            android:layout_height="@dimen/app_bar_height"
            android:fitsSystemWindows="true"
            android:theme="@style/AppTheme.AppBarOverlay">

```
        <android.support.design.widget.CollapsingToolbarLayout
            android:id="@+id/toolbar_layout"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:fitsSystemWindows="true"
            app:contentScrim="?attr/colorPrimary"
            app:layout_scrollFlags="scroll|exitUntilCollapsed">

            <ImageView
                android:onClick="onImageClick"
                android:id="@+id/productImage"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:scaleType="centerCrop"
                android:src="@drawable/placeholder_female"
                app:layout_collapseMode="parallax" />

            <android.support.v7.widget.Toolbar
                android:id="@+id/toolbar"
                android:layout_width="match_parent"
                android:layout_height="?attr/actionBarSize"
                app:layout_collapseMode="pin"
                app:layout_scrollFlags="scroll|enterAlways"
                app:popupTheme="@style/AppTheme.PopupOverlay">

            </android.support.v7.widget.Toolbar>

            <ImageView
                android:id="@+id/brandImage"
                android:layout_width="wrap_content"
                android:layout_height="?attr/actionBarSize"
                android:layout_gravity="bottom"
                android:layout_marginLeft="50dp"
                android:adjustViewBounds="true"
                android:maxWidth="150dp"
                android:scaleType="fitCenter" />


        </android.support.design.widget.CollapsingToolbarLayout>
    </android.support.design.widget.AppBarLayout>
```

`

**Comment by rogerlopz** [comment]
I'm having the same exact issue.
It happens with an `ImageZoom` inside `CollapsingToolbarLayout`

Edit:
This is my full layout

```
<?xml version="1.0" encoding="utf-8"?>
<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/consulta_de_punto"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:fitsSystemWindows="true"
    tools:context=".ConsultaDePuntoActivity">

    <android.support.design.widget.AppBarLayout
        android:id="@+id/app_bar"
        android:layout_width="match_parent"
        android:layout_height="@dimen/app_bar_height"
        android:fitsSystemWindows="true"
        android:theme="@style/AppTheme.AppBarOverlay">

        <android.support.design.widget.CollapsingToolbarLayout
            android:id="@+id/toolbar_layout"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:fitsSystemWindows="true"
            app:contentScrim="?attr/colorPrimary"
            app:layout_scrollFlags="scroll|exitUntilCollapsed">

            <com.github.oliveiradev.image_zoom.lib.widget.ImageZoom
                android:id="@+id/iv_act_consulta_punto_imagen_punto"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:fitsSystemWindows="true"
                android:largeHeap="true"
                android:scaleType="centerCrop"
                app:layout_collapseMode="parallax" />

            <android.support.v7.widget.Toolbar
                android:id="@+id/toolbar"
                android:layout_width="match_parent"
                android:layout_height="?attr/actionBarSize"
                app:layout_collapseMode="pin"
                app:popupTheme="@style/AppTheme.PopupOverlay" />

        </android.support.design.widget.CollapsingToolbarLayout>
    </android.support.design.widget.AppBarLayout>

    <android.support.v4.widget.SwipeRefreshLayout
        android:id="@+id/swipe_refresh_layout"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:layout_behavior="@string/appbar_scrolling_view_behavior">
        <include layout="@layout/content_consulta_de_punto" />
    </android.support.v4.widget.SwipeRefreshLayout>

    <android.support.design.widget.FloatingActionButton
        android:id="@+id/fab"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_margin="@dimen/fab_margin"
        app:layout_anchor="@id/app_bar"
        app:layout_anchorGravity="bottom|end"
        app:srcCompat="@drawable/ic_camera_white" />

</android.support.design.widget.CoordinatorLayout>
```

And this is the stacktrace:

```
FATAL EXCEPTION: main
                                                                          Process: mx.com.adsum.tm_ctrl_app, PID: 13834
                                                                          java.lang.ClassCastException: android.view.ContextThemeWrapper cannot be cast to android.app.Activity
                                                                              at com.github.oliveiradev.image_zoom.lib.widget.ImageZoom.performClick(ImageZoom.java:79)
                                                                              at android.view.View$PerformClick.run(View.java:20262)
                                                                              at android.os.Handler.handleCallback(Handler.java:815)
                                                                              at android.os.Handler.dispatchMessage(Handler.java:104)
                                                                              at android.os.Looper.loop(Looper.java:194)
                                                                              at android.app.ActivityThread.main(ActivityThread.java:5637)
                                                                              at java.lang.reflect.Method.invoke(Native Method)
                                                                              at java.lang.reflect.Method.invoke(Method.java:372)
                                                                              at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
                                                                              at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
```

**Comment by oliveiradev** [comment]
@rogerlopz I will try simulate using this layout and return a feedback

**Comment by oliveiradev** [comment]
fixed problem on version **0.2.1**
